src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx

```diff
@@ -231,10 +240,7 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
                       id='destinationFacilityId'
                       label={strings.TO_NURSERY_REQUIRED}
                       selectedValue={localRecords[0].destinationFacilityId?.toString()}
-                      options={getAllNurseries(organization).map((nursery) => ({
-                        label: nursery.name,
-                        value: nursery.id.toString(),
-                      }))}
+                      options={destinationNurseriesOptions}
```

Comment:
should this be a state or can it be a computed function like we do on line #230 
either is fine, just looking at consistency

Reply:
The problem here is that we need to change the options according to selectedNursery. I don't think the function will execute automatically every time that value change